packages/styles/scss/patterns/blocks/leadSpaceBlock/_leadspace-block.scss

```diff
@@ -0,0 +1,61 @@
+@import '../../../globals/imports';
```

Comment:
There's a lot of hardcoded values here. You should be using spacing tokens, etc. from carbon: https://www.carbondesignsystem.com/guidelines/spacing

Reply:
used variables for spacing.